components/Form/Password.js

```diff
@@ -1,56 +1,28 @@
 import React from "react";
-import { FormattedMessage, defineMessages, injectIntl, intlShape } from "react-intl";
+import { FormattedMessage, injectIntl } from "react-intl";
 import PropTypes from "prop-types";
-import cockpit from "cockpit";
-
-const messages = defineMessages({
-  weakPassword: {
-    defaultMessage: "Password is too weak"
-  },
-  rejectedPassword: {
-    defaultMessage: "Password is not acceptable"
-  }
-});
 
 class Password extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       passwordOne: "",
       passwordTwo: "",
-      displayWarningMatch: false,
-      passwordQuality: "",
-      warningQuality: "",
-      displayWarningQual: false
+      displayWarningMatch: false
     };
     this.handleChangePasswordOne = this.handleChangePasswordOne.bind(this);
     this.handleChangePasswordTwo = this.handleChangePasswordTwo.bind(this);
     this.handleWarnings = this.handleWarnings.bind(this);
-    this.validateQuality = this.validateQuality.bind(this);
-    this.passwordQualityFail = this.passwordQualityFail.bind(this);
-    this.passwordQualityPass = this.passwordQualityPass.bind(this);
   }
 
-  // password quality is only checked for passwordOne
-  // password match is checked on both
+  // password match is checked when entering a value in either field
   // messages only display during handleWarnings on blur
   // but will be removed during handleChange... when warnings are no longer true
 
   handleChangePasswordOne(event) {
     this.setState({
-      passwordOne: event.target.value,
-      displayWarningQual: false
+      passwordOne: event.target.value
     });
-    if (event.target.value !== "") {
-      this.validateQuality(event.target.value);
-      // if the password passes quality check, this function will also check
-      // if both values match, and will pass the password to the parent component
-    } else {
-      this.setState({
-        passwordQuality: "",
-        warningQuality: ""
-      });
-    }
     if (event.target.value === this.state.passwordTwo) {
       this.setState({ displayWarningMatch: false });
       if (event.target.value === "") {
```

Comment:
The Create button is not disabled when only the first password is entered. Adding the else statement will invalidate the password unless both passwords are empty.

![createButtonEnabled](https://user-images.githubusercontent.com/11712857/83872949-37d38080-a733-11ea-9754-b1034baa7b77.png)

Reply:
```suggestion
      if (event.target.value === "") {
        this.props.setValidPassword(event.target.value);
      }
    } else {
      this.props.setValidPassword();
    }
```